netctl/netctl.go

```diff
@@ -33,12 +33,12 @@ func createPolicy(ctx *cli.Context) {
 	tenant := ctx.String("tenant")
 	policy := ctx.Args()[0]
 
-	logrus.Infof("Creating policy %s:%s", tenant, policy)
-
 	errCheck(ctx, getClient(ctx).PolicyPost(&contivClient.Policy{
 		PolicyName: policy,
 		TenantName: tenant,
 	}))
+
+	fmt.Printf("Creating policy %s:%s\n", tenant, policy)
```

Comment:
In other functions also we are using log.Infof.
Check in createTenant and deleteTenant etc.